specviz/plugins/arithmetic/__init__.py

```diff
@@ -0,0 +1 @@
+from .arithmetic_editor import *
```

Comment:
New line needed

Reply:
Good catch. I went ahead and updated it to only import the two classes that make up the layer arithmetic!